### PR TITLE
Data generation

### PR DESCRIPTION
**Pull request by gingermostly**
I am currently using a while loop for generating my data and it's generating 10 million reservation records and 100k restaurants in around 53 seconds. I'm wondering if there would be a benefit to refactoring to promises, please let me know if you have any comments and/or if anything in my code doesn't make sense. Thanks!

### CODE REVIEW

File: database/seed.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
+const faker = require('faker');
+var fs = require('fs');
+
+var generateReservations = function() {
```
Comment (chagilguiab) [inline database/seed.js:4]: Nice work with the randomized data here to simulate "realistic" data

**Review by chagilguiab — COMMENTED** [review]
Nice work, Ginger. 
My main note is to add `npm-debug.log` and any CSV files to your `.gitignore`; I don't think you should be pushing those up.
re: refactoring to Promises. I used Promises and it seemed to actually _increase_ my script's run-time, so I'm hesitant to recommend it, though it _seems_ like a "cleaner" solution.

**Review by hectwilliams — COMMENTED** [review]
image
You may want to use **for loops** instead , for improved readability.

Everything else looks good! Great job!

File: database/seed.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+      tableNumber += 1;
+    }
+  }
+  return tables;
+}
+
+var writeReservations = function() {
+  console.time('timing seed');
+  var count = 1;
+  while (count <= 10) {
```
Comment (therobinkim) [inline database/seed.js:37]: +1 on recommendation to use for loops

File: database/seed.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    fs.writeFile(`./database/reservations-${count}.json`, JSON.stringify(generateReservations()), function(err){
```
Comment (therobinkim) [inline database/seed.js:38]: I think all 10 files get written in parallel. Is this what you intended for?

**Comment by therobinkim** [comment]
I would recommend setting up your eslint config for Airbnb's style guide asap to catch small style issues (eg, `var` vs `let` vs `const`, semicolons, commas, etc)

**Pull request merged**